recipe/meta.yaml

```diff
@@ -57,7 +57,7 @@ test:
     - pytest-console-scripts
     - pytest-cov
     - pytest-timeout
-    - pytest-tornasync
+    - pytest-jupyter >=0.4
```

Comment:
would be good to get https://github.com/conda-forge/pytest-jupyter-feedstock/pull/1

Reply:
[e4a7b99](https://github.com/conda-forge/jupyter_server-feedstock/pull/105/commits/e4a7b992b42cc56c7d0db9ef9e87d8fd0c78e832)